lib/rules/font-weight-notation/index.js

```diff
@@ -103,6 +103,16 @@ const rule = function(expectation, options) {
 			const weightValueOffset = decl.value.indexOf(weightValue);
 
 			if (expectation === 'numeric') {
+				if (decl.parent.type === 'atrule' && decl.parent.name.toLowerCase() === 'font-face') {
+					const weightValueNumbers = weightValue.match(/\d{1,}\.?\d+/g);
```

Comment:
I think better use `value` parser here instead regexp, because comments can be between values and it will be false positive

Reply:
`postcss.list.space()` might works as well. We have it in this rule:

https://github.com/stylelint/stylelint/blob/38d89fefe5569545e9239f7867491043d264dec0/lib/rules/font-weight-notation/index.js#L66

Reply:
Thank you for review @evilebottnawi and @hudochenkov ! The edge case that have to processed properly is
```css
font-weight: 400/* 700 */700;
```
In this case we get the ```weightValue``` equals to ```400700```. Probably we need to analyze the ```decl.raws``` data with ```postcss-value-parser``` in such case.

Reply:
I don't think `font-weight: 400/* 700 */700;` would be valid. Let's consider that we have at least one space before or after comment: `font-weight: 400 /* 700 */700;`.

Reply:
In this case I think ```list.space``` would be enough. I updated the PR.